### ISSUE TITLE
Problems upgrading to npm@3

### ISSUE THREAD

**Issue by uqee** [angular/protractor#3784]
I have a fully working setup with npm@2. If I upgrade npm to v3, remove `node_modules` and do a fresh `npm install`, `webdriver-manager update --standalone`, it stops working. By that I mean every test yields `A Jasmine spec timed out. Resetting the WebDriver Control Flow.`

Probably connected to the jasmine's issue: https://github.com/jasmine/jasmine-npm/issues/63, which was hopefully fixed at v2.5.2. Protractor v4.0.11 requires the older jasmine v2.4.1.

**Comment by sjelin** [comment]
I'm fairly certain I use npm@3 with no problem.  What happens when you force protractor to use a different version of jasmine with shrink-wrap?  Also, does this happen for any tests?

**Comment by uqee** [comment]
Another version of jasmine doesn't help. As well as npm@4.
It's not a critical bug though, but a strange one..

**Comment by juliemr** [comment]
We've been using npm@3 for a while successfully, and I haven't heard any other cases of this. @uqee can you reproduce on other machine? Can you provide a minimum case for us to take a look at?

**Comment by uqee** [comment]
Sure. I was expecting the minimum case question, will do that soon. Well, may be in a few weeks)

**Comment by uqee** [comment]
Didn't get any luck to make my setup work with NPM either v3 or v4, even with the cleanest default example from the [tutorial page](http://www.protractortest.org/#/tutorial), even on a clean another macbook, until I decided to just update all dependencies. Then some magic happened and it started to work as expected. To whom it may concern, I'll leave here the versions of my dependencies, which were changed.

Versions were:

```
"chromedriver": "^2.24.1",
"geckodriver": "^1.1.2",
"protractor": "^4.0.11",
"selenium-webdriver": "^3.0.0-beta-3",
```

Versions now:

```
"chromedriver": "^2.27.2",
"geckodriver": "^1.3.0",
"protractor": "^5.0.0",
"selenium-webdriver": "^3.0.1",
```
